tfhe/benches/core_crypto/ks_bench.rs

```diff
@@ -175,6 +175,101 @@ fn keyswitch<Scalar: UnsignedTorus + CastInto<usize> + Serialize>(
     }
 }
 
+fn packing_keyswitch<Scalar, F>(
+    criterion: &mut Criterion,
+    bench_name: &str,
+    parameters: &[(String, CryptoParametersRecord<Scalar>)],
+    ks_op: F,
+) where
+    Scalar: UnsignedTorus + CastInto<usize> + Serialize,
+    F: Fn(
+        &LwePackingKeyswitchKey<Vec<Scalar>>,
+        &LweCiphertextList<Vec<Scalar>>,
+        &mut GlweCiphertext<Vec<Scalar>>,
+    ),
+{
+    let bench_name = format!("core_crypto::{bench_name}");
+    let mut bench_group = criterion.benchmark_group(&bench_name);
+
+    // Create the PRNG
+    let mut seeder = new_seeder();
+    let seeder = seeder.as_mut();
+    let mut encryption_generator =
+        EncryptionRandomGenerator::<ActivatedRandomGenerator>::new(seeder.seed(), seeder);
+    let mut secret_generator =
+        SecretRandomGenerator::<ActivatedRandomGenerator>::new(seeder.seed());
+
+    for (name, params) in parameters.iter() {
+        let lwe_dimension = params.lwe_dimension.unwrap();
+        let glwe_dimension = params.glwe_dimension.unwrap();
+        let polynomial_size = params.polynomial_size.unwrap();
+        let ks_decomp_base_log = params.ks_base_log.unwrap();
+        let ks_decomp_level_count = params.ks_level.unwrap();
+        let ciphertext_modulus = params.ciphertext_modulus.unwrap();
```

Comment:
this is not correct, you could use the compression parameters from shortint to use here as they have packing keyswitch parameters

Reply:
Alright, but then the doctest example for the packing function is wrong no ? Since it other noise constraints (gaussian instead of t-uniform).

Reply:
we'll discuss that, not sure what this is about